### PR TITLE
enhance: import supports null in parquet and json formats

### PR DESCRIPTION
**Pull request by smellthemoon**
#31728

### CODE REVIEW

File: internal/util/importutilv2/json/row_parser.go [hunk 1]
```diff
@@ -401,6 +404,147 @@ func (r *rowParser) parseEntity(fieldID int64, obj any) (any, error) {
 	}
 }
 
+func (r *rowParser) parseNullableEntity(fieldID int64, obj any) (any, error) {
```
Comment (tedxu) [inline internal/util/importutilv2/json/row_parser.go:407]: This piece of code looks repetitive with `parseEntity`. Is it possible that we merge these two functions together?

**Review by tedxu — COMMENTED** [review]
/lgtm

File: internal/util/importutilv2/json/row_parser.go (hunk 1 above)
Comment (smellthemoon) [reply internal/util/importutilv2/json/row_parser.go:407]: I write separately to make logic more clear. but it's ok to combine it. I will do it in next pr.

**Pull request merged**